tests/test_container.py

```diff
@@ -1,7 +1,6 @@
 from unittest import TestCase
 
-from wrapper_writer.container import Container
```

Comment:
Is it also worth combining the test_container.py test_method.py into one test_code_elements.py?

Reply:
tests combined